docs/ops/state/checkpoints.md

```diff
@@ -63,6 +63,24 @@ files. The meta data file and data files are stored in the directory that is
 configured via `state.checkpoints.dir` in the configuration files, 
 and also can be specified for per job in the code.
 
+The current checkpoint directory layout which introduced by FLINK-8531 is as follows:
+
+{% highlight yaml %}
+/user-defined-checkpoint-dir
+    |
+    + --shared/
+    + --taskowned/
+    + --chk-00001/
+    + --chk-00002/
+    + --chk-00003/
+    ...
+{% endhighlight %}
+
+The **SHARED** directory is for state that is possibly part of multiple checkpoints, **TASKOWNED** is for state that must never by dropped by the JobManager, and **EXCLUSIVE** is for state that belongs to one checkpoint only. 
```

Comment:
The description order is according to the order of the above checkpoint directory layout.